<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDeepMRSegControls</class>
 <widget class="QWidget" name="QDeepMRSegControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxInstructions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>Instructions</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>• Load at least one image.
• Select Parameters.
• Run.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_DataSelection">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>XY Width</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string># classes</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Prep</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Ressize</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Reorient</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>idCol</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ROI</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QmitkDataStorageComboBox" name="m_cbReferenceNodeSelector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Model Directory</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>sList</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Rescale Norm</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Verb</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_PatientImage">
        <property name="text">
         <string>Reference Image</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>nJobs</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEdit_7"/>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEdit_8"/>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEdit_9"/>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="lineEdit_10"/>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="lineEdit_11"/>
      </item>
      <item row="12" column="1">
       <widget class="QLineEdit" name="lineEdit_12"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Run">
     <property name="title">
      <string>Generate Segmentation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRun">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonRunScript">
     <property name="text">
      <string>Run Script</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>466</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">QmitkDataStorageComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../InteractiveSegmentation/Plugins/upenn.cbica.captk.interactivesegmentation/resources/captkinteractivesegmentation.qrc"/>
  <include location="../../../../Modules/CbicaDeepMRSeg/resources/QExamplePython.qrc"/>
 </resources>
 <connections/>
</ui>
